src/chunkserver/network_worker_thread.cc

```diff
@@ -887,6 +887,18 @@ void worker_hdd_list_v2(csserventry *eptr, const uint8_t *data,
 	hddSerializeAllDiskInfosV2(ptr); // unlock
 }
 
+void worker_list_disk_groups(csserventry *eptr,
+                             [[maybe_unused]] const uint8_t *data,
+                             [[maybe_unused]] uint32_t length) {
+	TRACETHIS();
+
+	std::string diskGroups = hddGetDiskGroups();
+	static constexpr uint8_t kPadding = 5;
```

Comment:
Please, consider adding a little comment here about why value 5 was selected here to kPadding variable

Reply:
I have the same doubt :sweat_smile: 
Why adding `kPadding value = 5` to `diskGroups.size()` is needed?

Reply:
I renamed the variable to be kSerializedSizePlusNullChar. 5 was needed by the serialize function:
![image](https://github.com/user-attachments/assets/5bc284fd-7101-40bc-a753-a2e6d6804f79)